src/esp/assets/ResourceManager.cpp

```diff
@@ -165,8 +168,8 @@ void ResourceManager::initPhysicsManager(
   // if the desired simulator is not supported reset to "none" in metaData
   if (defaultToNoneSimulator) {
     physicsManagerAttributes->setSimulator("none");
-    physicsManager.reset(
-        new physics::PhysicsManager(*this, physicsManagerAttributes));
+    physicsManager = std::make_shared<physics::PhysicsManager>(
```

Comment:
@jturner65 @aclegg3  Do we want the same behavior down here with the reset?

Reply:
Yeah, whatever we end up doing should be the same in both places.